configurations/s02_pipeline_configuration.py

```diff
@@ -0,0 +1,201 @@
+from core_data_modules.cleaners import swahili
+from dateutil.parser import isoparse
+
+from src.pipeline_configuration_spec import *
+
+
+PIPELINE_CONFIGURATION = PipelineConfiguration(
+    pipeline_name="WUSC-LEAP-II",
+    project_start_date=isoparse("2021-03-01T10:30:00+03:00"), #Todo update before production
+    project_end_date=isoparse("2100-01-01T00:00:00+03:00"),
+    engagement_database=EngagementDatabaseClientConfiguration(
+        credentials_file_url="gs://avf-credentials/avf-engagement-databases-firebase-credentials-file.json",
+        database_path="engagement_databases/WUSC_KAKUMA_KALOBEYEI"
```

Comment:
Just checking we're happy for this to be WUSC-specific rather than location specific e.g. WUSC_KAKUMA_KALOBEYEI vs. KAKUMA_KALOBEYEI

Reply:
Good suggestion, I think we want it as KAKUMA_KALOBEYEI in case we run other none Wusc projects in the camps.